internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -34,11 +34,17 @@ var (
 // Server is a fake S2A Server for testing.
 type Server struct {
 	s2av2pb.UnimplementedS2AServiceServer
+	validToken string
+	isClientSide bool
```

Comment:
nit: could we call this variable "isAssistingClientSide" instead?

Rationale: the fake S2A assists a client or server, but the fake S2A itself is always acting as a server.